qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -210,6 +173,11 @@ def randomized_benchmarking_seq(nseeds=1, length_vector=None,
         npurity: the number of purity rb circuits (per seed)
         which equals to 3^n, where n is the dimension
     """
+    # Set modules (default is Clifford)
+    if group_gates is None or 'Clifford':
+        Gutils = clutils
+        Ggroup = Clifford
+
```

Comment:
put an error handler in here to catch unsupported groups

Reply:
Done.